src/install.js

```diff
@@ -22,6 +22,14 @@ function getPlatform() {
     return osName && arch && `${osName}_${arch}`;
 }
 
+function needsDefaultVersion(versionSpec) {
```

Comment:
Maybe more apt?

```suggestion
function isLatest(versionSpec) {
```

Reply:
You're right, that's more self-explanatory ! I will update that.